packages/ui/Alert/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "version": "0.0.0",
+  "name": "@cfxjs/alert",
+  "type": "module",
+  "main": "index.js"
```

Comment:
also make these package private if we don't want to publish them for now

the yarn versions file only prevent it from being published if the version is not changed

since the package is not published yet, ci will still publish them if they are not private package

Reply:
Agree, currently we make the package private.